lib/activerecord-multi-tenant/multi_tenant.rb

```diff
@@ -5,8 +5,13 @@ class Current < ::ActiveSupport::CurrentAttributes
     attribute :tenant
   end
 
-  def self.tenant_klass_defined?(tenant_name)
-    !!tenant_name.to_s.classify.safe_constantize
+  def self.tenant_klass_defined?(tenant_name, options = {})
+    class_name = if options[:class_name].present?
```

Comment:
rubocop doesn't like this indentation. `rubocop -A` should autocorrect it

Reply:
I fixed it and pushed.